hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -911,6 +900,22 @@ boolean evictBucketEntryIfNoRpcReferenced(BlockCacheKey blockCacheKey, BucketEnt
     return false;
   }
 
+  @Override
+  public void onConfigurationChange(Configuration config) {
+    this.acceptableFactor = conf.getFloat(ACCEPT_FACTOR_CONFIG_NAME, DEFAULT_ACCEPT_FACTOR);
```

Comment:
Haven't touch this area for a long time, all these configs can be loaded dynamically? IIRC, BucketCache will pre alloc all the memory chunks when initializing, configuration refreshing can not trigger the allocation again.

Reply:
nit: can we add a comment of javadoc on what properties could not be dynamic config? basically we changed `ConfigurationObserver` to be part of the dynamic config.